components/esp_mqtt_cxx/examples/tcp/pytest_mqtt_tcp.py

```diff
@@ -0,0 +1,4 @@
+# SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+# SPDX-License-Identifier: Unlicense OR CC0-1.0
+def test_dummy() -> None:
```

Comment:
Cleanup?

Reply:
No, this is necessary to have the examples detected as test apps and built.